corehq/apps/analytics/static/analytix/js/gtx.js

```diff
@@ -63,15 +61,58 @@ hqDefine('analytix/js/gtx', [
         });
     };
 
+    /**
+     * Modified version of utils.initApi to enable GTM on the India environment.
+     * This does not checks for `global.isEnabled` and instead directly checks for the environment.
+     * This is done to avoid enabling other analytics tooling on the India environment.
```

Comment:
> avoid enabling other analytics tooling

Do we know what these are and why we don't want them enabled for india?

Reply:
- Google Analytics (Marketing)
- Kissmetrics
- Hubspot
- Drift
- Appcues

These are currently not enabled for India environment (for UI events tracking) and we wish to keep the same state. Also we do not know the reason why these tools were not enabled for India in the first place, so enabling only what we need (Tag Manager) in this change.

Reply:
FYI -  I say for UI events tracking because there are some background tasks (hubspot & Kissmetrics) that runs on all environments.

Reply:
Thanks @ajeety4 

thanks for the info.

I am a bit uneasy about the js overrite that makes this hard to maintain though I can be okay if this is temporary.
Going to offline.

This is non blocking, so feel free to go ahead with the PR.